src/main/java/com/faforever/api/error/ErrorCode.java

```diff
@@ -71,7 +71,12 @@
   USERNAME_RESERVED(162, "Invalid account data", "The username ''{0}'' can only be claimed by the original owner within {1} months after it has been freed."),
   UNKNOWN_IDENTIFIER(163, "Unable to resolve user", "The identifier does neither match a username nor an email: {0}"),
   ALREADY_REGISTERED(164, "Registration failed", "You can't create a new account because you already have one."),
-  STEAM_LINK_NO_FA_GAME(141, "Linking to Steam failed", "You do not own Forged Alliance on Steam or your profile is private.");
+  STEAM_LINK_NO_FA_GAME(165, "Linking to Steam failed", "You do not own Forged Alliance on Steam or your profile is private."),
+  FILE_NAME_TOO_LONG(166, "Invalid file name", "The file name must not exceed {0} characters, was: {1}"),
+  FILE_SIZE_EXCEEDED(167, "Invalid file size", "The file size must not exceed {0} bytes, was: {1}"),
```

Comment:
The `{0}` should be `''{0}''` **however**, since it's a number, this is not needed **if** we can make sure that a number needs to be passed. For that, we can use `{0,number}` :)

Please also fix it at other places above, where I didn't do it.